docs/source/api/v4/servers_id.rst

```diff
@@ -25,6 +25,7 @@ Allow user to edit a server.
 
 :Auth. Required: Yes
 :Roles Required: "admin" or "operations"
+:Permissions Required: SERVER:UPDATE, SERVER:READ, CACHE-GROUP:UPDATE, DELIVERY-SERVICE:UPDATE, CDN:UPDATE, SERVER:READ, DELIVERY-SERVICE:READ, CDN:READ, PHYSICAL-LOCATION:READ, CACHE-GROUP:READ, TYPE:READ, PROFILE:READ
```

Comment:
Same as above RE: `DELIVERY-SERVICE:UPDATE`, `CACHE-GROUP:UPDATE`, and `CDN:UPDATE`.